docker/lae.nix

```diff
@@ -0,0 +1,337 @@
+{ pkgs ? import <nixpkgs> {}
+, pythonPackages ? import <nixpkgs.python27Packages> {}
```

Comment:
This can be expressed as `pythonPackages ? pkgs.python27Packages` to minimize the risk that these drift apart.